hardware/Power.kicad_sch

```diff
@@ -0,0 +1,5378 @@
+    )
+    (property "D1PN" "311-1.00KHRCT-ND" (id 5) (at 170.4086 89.0016 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MFN" "Yageo" (id 6) (at 170.4086 89.0016 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MPN" "RC0603FR-071KL" (id 7) (at 170.4086 89.0016 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Tolerance" "1%" (id 8) (at 170.4086 89.0016 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+  )
+
+  (symbol (lib_id "Device:R_Small") (at 229.87 52.07 0) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid "00000000-0000-0000-0000-00005d658377")
+    (property "Reference" "R24" (id 0) (at 231.3686 51.1302 0)
+      (effects (font (size 0.9906 0.9906)) (justify left))
+    )
+    (property "Value" "1M" (id 1) (at 231.3686 52.9844 0)
+      (effects (font (size 0.9906 0.9906)) (justify left))
+    )
+    (property "Footprint" "Resistor_SMD:R_0603_1608Metric" (id 2) (at 229.87 52.07 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "" (id 3) (at 229.87 52.07 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "D1N" "Digikey" (id 4) (at 231.3686 48.5902 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "D1PN" "311-1.00MHRCT-ND" (id 5) (at 231.3686 48.5902 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MFN" "Yageo" (id 6) (at 231.3686 48.5902 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MPN" "RC0603FR-071ML" (id 7) (at 231.3686 48.5902 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Tolerance" "1%" (id 8) (at 231.3686 48.5902 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+  )
+
+  (symbol (lib_id "Device:R_Small") (at 229.87 59.69 0) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid "00000000-0000-0000-0000-00005d658b9f")
+    (property "Reference" "R29" (id 0) (at 234.95 58.42 0)
+      (effects (font (size 0.9906 0.9906)) (justify right))
+    )
+    (property "Value" "91k" (id 1) (at 234.95 60.325 0)
+      (effects (font (size 0.9906 0.9906)) (justify right))
+    )
+    (property "Footprint" "Resistor_SMD:R_0603_1608Metric" (id 2) (at 229.87 59.69 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "" (id 3) (at 229.87 59.69 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "D1N" "Digikey" (id 4) (at 233.4768 56.2102 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "D1PN" "311-91.0KLRCT-ND" (id 5) (at 233.4768 56.2102 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MFN" "Yageo" (id 6) (at 233.4768 56.2102 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "MPN" "RC0402FR-0791KL" (id 7) (at 233.4768 56.2102 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Tolerance" "1%" (id 8) (at 233.4768 56.2102 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+  )
```

Comment:
Apologies, this is caught up in the file update (probably should've split this up but it was a minor change).